src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < 6; i++) {
+            Figure figure;
+            if (i < 6 / 2) {
```

Comment:
use .length of array instead of hardcoding

Reply:
fixed